operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -43,14 +43,9 @@ kiali_defaults:
       in_cluster: true
       password: ""
       service: "grafana"
-      service_dashboard_pattern: "Istio%20Service%20Dashboard"
-      service_namespace: ""
+      namespace: ""
```

Comment:
same as above - move "namespace" up so it is alphabetically listed (after in_cluster, before password).

Reply:
done